components/raftstore/src/store/fsm/store.rs

```diff
@@ -1554,12 +1555,19 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
             None
         };
 
+        let wal_sync_worker = Worker::new("wal-sync-worker");
+        let kv_clone = engines.kv.clone();
+        wal_sync_worker.spawn_interval_task(Duration::from_millis(100), move || {
+            kv_clone.flush_wal().unwrap();
```

Comment:
We are seeing segmentation fault during init. I am wondering could it because of rocksdb not initialized before flust_wal() is called.

Reply:
the rocksdb should be already initialized before starting the worker